src/Pact/Analyze/Feature.hs

```diff
@@ -499,6 +507,86 @@ doc FModulus = Doc
         (TyCon int)
   ]
 
+-- Bitwise operators
+
+doc FBitwiseAnd = Doc
```

Comment:
The feature system produced the [Property and Invariant Functions](https://pact-language.readthedocs.io/en/stable/pact-properties-api.html) documentation.